packages/grant-explorer/src/features/discovery/LandingHero.tsx

```diff
@@ -14,11 +15,15 @@ export default function LandingHero() {
   );
 }
 
+const numberFormatter = new Intl.NumberFormat();
 const ByTheNumbers = () => {
   // TODO: fetch data (where to get this?)
-  const uniqueDonations = "3.8M";
-  const raisedFunds = "3,715";
-  const fundingDistributed = "$50,000,000+";
+  const uniqueDonations = useAnimateNumber({ value: Number(3800) });
```

Comment:
Is there a particular reason to use the Number constructor here, and not just a number literal?

Reply:
No, you're right. It should be just `3800`.